src/components/card-grid/card-grid.jsx

```diff
@@ -13,8 +13,8 @@ export default class CardGrid extends React.Component {
     this.state = this.modelInitStateFromDataProp(props.data);
   }
 
-  componentWillReceiveProps(nextProps) {
-    const state = this.modelInitStateFromDataProp(nextProps.data);
+  UNSAFE_componentWillReceiveProps(nextProps) { // eslint-disable-line camelcase
```

Comment:
[2 in a row](https://github.com/ProvidenceGeeks/website-frontend/pull/186#discussion_r334243106) 😮 

Am I missing something?  🤔

Reply:
https://github.com/ProvidenceGeeks/website-frontend/pull/186#discussion_r334725592